src/ui/server/controllers/insurance/export-contract/how-much-the-agent-is-charging/validation/index.ts

```diff
@@ -0,0 +1,7 @@
+import validationRules from './rules';
+import combineValidationRules from '../../../../../helpers/combine-validation-rules';
+import { RequestBody, ValidationErrors } from '../../../../../../types';
+
+const validation = (formBody: RequestBody): ValidationErrors => combineValidationRules(validationRules, formBody) as ValidationErrors;
```

Comment:
do we need documentation here to be consistent?

Reply:
There are around ~30+ instances of this, can we tackle in one PR?